data/low_income_thresholds.json

```diff
@@ -154,6 +154,34 @@
       }
     }
   },
+  "IL": {
+    "default": {
+      "source_url": "https://dceo.illinois.gov/communityservices/homeweatherization.html",
+      "thresholds": {
+        "1": 29160,
+        "2": 39440,
+        "3": 49720,
+        "4": 60000,
+        "5": 70280,
+        "6": 80560,
+        "7": 90840,
+        "8": 93297
+      }
+    },
+    "il-state-of-illinois": {
```

Comment:
Since they're the same, I would keep `default` and delete this one, to stay with the convention in other states.